internal/executor/job/submit.go

```diff
@@ -53,7 +53,7 @@ type FailedSubmissionDetails struct {
 
 func (submitService *SubmitService) SubmitJobs(jobsToSubmit []*SubmitJob) []*FailedSubmissionDetails {
 	return submitService.submitJobs(jobsToSubmit)
-}
+} 
```

Comment:
Please remove this change - there shouldn't be extra whitespace at the end of lines. (I think running `golangci-lint` will warn about things like this, and using `go fmt file_xxx.go` will fix it for you.

Reply:
Thanks @richscott I will make these changes